_includes/desktop-install.html

```diff
@@ -0,0 +1,103 @@
+<div class="panel-group" id="accordion" role="tablist" aria-multiselectable="true">
+    <div class="panel panel-default">
+      <div class="panel-heading" role="tab" id="headingOne">
+        <h5 class="panel-title">
+          <a role="button" data-toggle="collapse" data-parent="#accordion" href="#collapseOne" aria-expanded="true" aria-controls="collapseOne">
+            Windows
+          </a>
+        </h5>
+      </div>
+      <div id="collapseOne" class="panel-collapse collapse" role="tabpanel" aria-labelledby="headingOne">
+        <div class="panel-body">
+          <a class="btn btn-primary" href="https://desktop.docker.com/win/main/amd64/Docker%20Desktop%20Installer.exe?utm_source=docker&utm_medium=webreferral&utm_campaign=docs-driven-download-win-amd64" role="button">Download file</a> 
+        <br>
+        <br>
+        <b>Checksum:</b> fe430d19d41cc56fd9a4cd2e22fc0e3522bed910c219208345918c77bbbd2a65
```

Comment:
We need to say it’s SHA-256 somehow, e.g.
```suggestion
        <b>Checksum:</b>SHA-256 fe430d19d41cc56fd9a4cd2e22fc0e3522bed910c219208345918c77bbbd2a65
```
or
```suggestion
        <b>Checksum:</b>SHA-256-fe430d19d41cc56fd9a4cd2e22fc0e3522bed910c219208345918c77bbbd2a65
```

Reply:
I think the first option looks better - will make the ammendments

Reply:
Does SHA-256 need to be added to all of them or just windows?

Reply:
It’s all of them.